File: problem-solving/codewars/EvenorOdd.py
# Even or Odd


def even_or_odd(number):
    if (number % 2 == 0):
        return "Even"
    else:
        return "Odd"


def main():
    number = int(input())

    print(even_or_odd(number))


if __name__ == '__main__':
    main()

File: codewars/rust/old/even_or_odd.rs
// https://www.codewars.com/kata/53da3dbb4a5168369a0000fe/train/rust


fn even_or_odd(i:i32) -> &'static str {
	if i % 2 == 0 {
		"Even"
	} else {
		"Odd"
	}
}

fn main() {
	let mut line = String::new();

	std::io::stdin().read_line(&mut line).expect("X");
	let i:i32 = line.trim().parse().expect("X");

	println!("{}", even_or_odd(i));
}
